Source/UI/Views/AddressView.swift

```diff
@@ -23,8 +23,7 @@ public class AddressView: UIView {
     let countrySelectionViewController = CountrySelectionViewController()
     let countryRegionTapGesture = UITapGestureRecognizer()
 
-    let doneButton = UIBarButtonItem(barButtonSystemItem: UIBarButtonItem.SystemItem.done,
-                                     target: self, action: nil)
+    let doneButton = UIBarButtonItem(barButtonSystemItem: UIBarButtonItem.SystemItem.done, target: nil, action: nil)
```

Comment:
Didn't check this one, but would imagine its also not used, as the one from AddressVC?

Reply:
When New UI is fully approved, I will ask Patrick to remove it.

Reply:
I am not sure why this can't be removed now. Removing the warning is not helpful at all as compiler is telling us this is implemented wrong and we're just telling it to shut up, whilst 

Using the autocorrect from Xcode only hides the warning whilst allowing the bad configuration to continue existing. Safe solutions imo would be:
- removing the button if not needed. This way no-one can forget about it and strip one useless property
- ensuring the property is created **after** the class has been initialised, when `self` actually exists, and probably referencing the `action` as well. I am pretty confident the reason action was nil is that compiler would refuse to compile as self was not available and after many hacky attempts to get it building, that was the committed solution. 4 years later we can fix it if we see it.

Also, there's a reason that property lies there unused, and one of the potential sources is someone saying _I'll ask x to do it later_.

Reply:
`TODO` is added to check the logic for CardViewController. 
for now, it is a warning fix and it's working.